addons/l10n_pl_sale_stock/models/account_move.py

```diff
@@ -0,0 +1,18 @@
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_pl_delivery_date = fields.Date(
+        compute='_compute_l10n_pl_delivery_date', store=True,
```

Comment:
I planned on doing it during the fw-port of this, as `delivery_date` is in account in 16.4.
So, if I wait until master, we would have doubles with this.
It's not ideal as we want to generally merge it as fast as possible, but I think it would be for the best

Reply:
Ooops, indeed, I looked at the task a bit too quickly ; I thought it wasn't merged yet :o But indeed, better do that in master; as you'll need a migration script. It's just one saas with a "duplicate" field; no big deal.